server/datastore/datastore_entities.go

```diff
@@ -134,52 +134,55 @@ type SumoLogicConfig struct {
 }
 
 const (
```

Comment:
Due to my "OCD," I had to sort these data stores alphabetically to find our providers better.
Also, I fixed a typo where some data stores had the constant starting with `Datastore` instead of `DataStore`.